cluster/charts/crossplane/templates/deployment.yaml

```diff
@@ -212,13 +212,11 @@ spec:
           - name: {{ $key | replace "." "_" }}
             value: {{ $value | quote }}
         {{- end}}
-        {{- if .Values.packageManager.enableAutomaticDependencyDowngrade }}
-          - name: "AUTOMATIC_DEPENDENCY_DOWNGRADE_ENABLED"
-            value: "true"
-        {{- end }}
```

Comment:
This is now just a feature flag like any other. See updated `pkg_test.go` E2E for example.